terraform_egress_pub/scripts/egress_pub.py

```diff
@@ -51,18 +51,10 @@
 # The following CIDR blocks are where our Qualys web application scans
 # originate from:
 qualys_was_ips = (
-    "64.39.99.19/32",
-    "64.39.99.20/30",
-    "64.39.99.24/31",
-    "64.39.99.98/31",
-    "64.39.99.100/32",
-    "64.39.99.131/32",
-    "64.39.99.132/30",
-    "64.39.99.136/29",
-    "64.39.99.144/31",
     "64.39.108.103/32",
-    "64.39.108.104/29",
+    "64.39.108.104/29", 
     "64.39.108.112/28",
+    "64.39.99.0/24"
```

Comment:
This should take care of the pre-commit issues:
```suggestion
    "64.39.108.104/29",
    "64.39.108.112/28",
    "64.39.99.0/24",
```